src/sentry/static/sentry/app/utils/theme.tsx

```diff
@@ -301,6 +301,9 @@ const theme = {
     empty: '#e2dee6',
     colors: ['#ec5e44', '#f38259', '#f9a66d', '#98b480', '#57be8c'],
   },
+
+  // Needed for @rebass
+  space: [0, 8, 16, 20, 30],
```

Comment:
Is this intended to match the scale used in the space() utility? If so could we make that function use this data?

Reply:
Ah it is, but this doesn't belong in this PR